collect_app/src/main/java/org/odk/collect/android/injection/config/AppDependencyModule.java

```diff
@@ -135,11 +135,11 @@ WebCredentialsUtils provideWebCredentials() {
     }
 
     @Provides
-    ServerFormsDetailsFetcher provideDownloadFormListDownloader(
+    FormListDownloader formListDownloader(
```

Comment:
It felt like a good idea to bring this back as the `ServerPollingJob` and `DownloadFormListTask` were both using it and don't have a lot of tests around them. This would make it riskier to iterate on and improve the `ServerFormsDetailsFetcher`.

Reply:
I'll deprecate in a later commit.